docs/source/user_guide/export_datasets.rst

```diff
@@ -47,13 +47,50 @@ a |DatasetView| into any format of your choice via the basic recipe below.
 
         # Export the dataset!
         dataset_or_view.export(
-            export_dir=export_dir, dataset_type=dataset_type, label_field=label_field
+            export_dir=export_dir,
+            dataset_type=dataset_type,
+            label_field=label_field,
         )
 
     Note the `label_field` argument in the above example, which specifies the
-    particular label field that you wish to export. This is necessary your
+    particular label field that you wish to export. This is necessary if your
     FiftyOne dataset contains multiple label fields.
 
+    The :meth:`export() <fiftyone.core.collections.SampleCollection.export>`
+    method also provides additional parameters that you can use to configure
+    the export, such as the `export_media` parameter, which allows you to
+    configure whether to copy, move, symlink, or omit the media files from
+    the export:
+
+    .. code-block:: python
+        :linenos:
+
+        #
+        # Variation #1
+        #
+        # Export the labels in the `ground_truth` field in COCO format, and
+        # write a `data.json` file describing the locations of the source media
+        # rather than exporting the actual media
+        #
+        dataset_or_view.export(
+            export_dir="/path/for/export",
+            dataset_type=fo.types.COCODetectionDataset,
+            label_field="ground_truth",
+            export_media=False,
```

Comment:
Shouldn't this be `export_media="manifest"` if you say it is writing a `data.json` file?

Reply:
yep you're correct; forgot to update this documentation 😅